dev-packages/e2e-tests/test-applications/nuxt-3/tests/performance.client.test.ts

```diff
@@ -0,0 +1,47 @@
+import { expect, test } from '@playwright/test';
+import { waitForTransaction } from '@sentry-internal/test-utils';
+import { SEMANTIC_ATTRIBUTE_SENTRY_OP, SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN } from '@sentry/core';
+
+test('sends a server action transaction on pageload', async ({ page }) => {
+  const transactionPromise = waitForTransaction('nuxt-3', transactionEvent => {
+    return transactionEvent.transaction.includes('GET /test-param/');
+  });
+
+  await page.goto('/test-param/1234');
+
+  const transaction = await transactionPromise;
+
+  expect(transaction.contexts.trace).toEqual(
+    expect.objectContaining({
+      data: expect.objectContaining({
+        [SEMANTIC_ATTRIBUTE_SENTRY_OP]: 'http.server',
+        [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: 'auto.http.otel.http',
+      }),
+    }),
+  );
+});
+
+test('does not send transactions for build asset folder "_nuxt"', async ({ page }) => {
+  let buildAssetFolderOccurred = false;
+
+  waitForTransaction('nuxt-3', transactionEvent => {
+    if (transactionEvent.transaction.includes('/_nuxt/')) {
+      buildAssetFolderOccurred = true;
+    }
+    return false; // expects to return a boolean (but not relevant here)
+  });
+
+  const transactionEventPromise = waitForTransaction('nuxt-3', transactionEvent => {
+    return transactionEvent.transaction.includes('GET /test-param/');
+  });
+
+  await page.goto('/test-param/1234');
+
+  const transactionEvent = await transactionEventPromise;
+
+  expect(buildAssetFolderOccurred).toBe(false);
+
+  // todo: url not yet parametrized
+  expect(transactionEvent.transaction).toBe('GET /test-param/1234');
```

Comment:
I am not sure why this is the case here - it shows `/test-param/*` in the UI but `1234` here. Relay seems to pick that up, but shouldn't the SDK as well?

Reply:
Isn't that because we aren't hitting relay, but our tunnel?